ui/src/components/home/ExperienceBiblicalCommunity.js

```diff
@@ -0,0 +1,468 @@
+                        process.env.PUBLIC_URL + '/images/home/community-5.png'
+                      }
+                      alt="Image 5"
+                      borderRadius="md"
+                      px={['0.5em', '2em']}
+                    />
+                  </Slider>
+                </Box>
+                {isMobile ? (
+                  <VStack>
+                    <VStack w="100%">
+                      <Flex w="100%">
+                        <HStack py="1em">
+                          <Box
+                            w="1.2em"
+                            h="1.2em"
+                            borderRadius="full"
+                            bg="black"
+                            display="flex"
+                            justifyContent="center"
+                            fontFamily="DMSerifDisplay_Regular"
+                            fontWeight="400"
+                            alignItems="center"
+                            color="white"
+                            fontSize="42"
+                          >
+                            1
+                          </Box>
+                          <Text
+                            color="black"
+                            fontSize="32"
+                            px="0.5em"
+                            fontFamily="DMSerifDisplay_Italic"
+                            fontWeight="400"
+                            lineHeight="39.48px"
+                            wordBreak="break-word"
+                          >
+                            Join a LIFE Group
+                          </Text>
+                        </HStack>
+                      </Flex>
+                      <Box>
+                        <Text
+                          color="#0B0F19"
+                          fontSize="14"
+                          fontFamily="Manrope"
+                          fontWeight="400"
+                          wordBreak="break-word"
+                        >
+                          There is no better way to get a taste of who we are
+                          and what we believe in then to check out one of our
+                          LIFE Groups. This is an opportunity to experience the
+                          life-changing power of Biblical community with Love,
+                          Investment, Faith, and Enjoyment.
+                        </Text>
+                      </Box>
+                      <Link
+                        href="https://hongkong.hmcc.net/forms/669bcf723858791ed22578ea"
+                        w="100%"
+                      >
+                        <Button
+                          w="100%"
+                          pl={10.5}
+                          pr={7}
+                          borderRadius={80}
+                          border="0.70px solid #D46764"
+                          justify="space-between"
+                          align="center"
+                          bgColor="#D46764"
```

Comment:
<img width="291" alt="image" src="https://github.com/user-attachments/assets/a5b690de-3f91-4bdc-951d-93a03e42e8df">

gray background in button. this `bgColor` should be nothing/same as container background. also bigger arrow (use `boxSize`) and also wider gap between the button contents (can increase button width i think?)

<img width="442" alt="image" src="https://github.com/user-attachments/assets/51992296-be55-40af-b33c-fe07a86e2115">

Reply:
<img width="376" alt="image" src="https://github.com/user-attachments/assets/7bfd8950-fac3-4025-80d4-b293762cde97">
also the button has a weird gray effect after being clicked on mobile.

i suspect you need to add a custom `_hover={{css:}}`. can see the chakra docs for this/see my events PR